### PR TITLE
feat: add depolyment information for Polygon Amoy

### PR DESCRIPTION
**Pull request by bmzig**
Polygon Mumbai will be deprecated in the near future, and there have been requests to deploy our contracts to Polygon Amoy. This commit contains all the information about our contract deployments to the Polygon Amoy/Sepolia test networks.

### CODE REVIEW

File: deploy/consts.ts [hunk 1]
```diff
@@ -270,5 +279,5 @@ export const CIRCLE_DOMAIN_IDs: { [chainId: number]: number } = {
   420: 2, // Optimism Goerli
   421613: 3, // Arbitrum Goerli
   84531: 6, // Base Goerli
-  80001: 7, // Polygon Mumbai
+  80001: 7, // Polygon Amoy
```
Comment (pxrl) [inline deploy/consts.ts:282]:
```suggestion
  80002: 7, // Polygon Amoy
```

**Review by pxrl — APPROVED** [review]
LGTM after one minor typo :+1:

File: deploy/consts.ts [hunk 2]
```diff
@@ -114,14 +114,14 @@ export const L1_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
...
-    polygonRootChainManager: "0xA0c68C638235ee32657e8f720a23ceC1bFc77C77",
-    polygonFxRoot: "0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2",
-    polygonERC20Predicate: "0x40ec5B33f54e0E8A33A975908C5BA1c14e5BbbDf",
-    polygonRegistry: "0x33a02E6cC863D393d6Bf231B697b82F6e499cA71",
-    polygonDepositManager: "0x401F6c983eA34274ec46f84D70b31C151321188b",
+    // https://github.com/maticnetwork/static/blob/master/network/testnet/amoy/index.json
+    polygonFxRoot: "0x0E13EBEdDb8cf9f5987512d5E081FdC2F5b0991e",
+    polygonERC20Predicate: "0x4258C75b752c812B7Fa586bdeb259f2d4bd17f4F",
+    l2WrappedMatic: "0x360ad4f9a9A8EFe9A8DCB5f461c4Cc1047E1Dcf9",
+    polygonRootChainManager: "0x34F5A25B627f50Bb3f5cAb72807c4D4F405a9232",
```
Comment (james-a-morris) [inline deploy/consts.ts:121]: Nit: could we keep these in the same ordering

File: deploy/consts.ts [hunk 3]
```diff
@@ -270,5 +278,5 @@ export const CIRCLE_DOMAIN_IDs: { [chainId: number]: number } = {
   420: 2, // Optimism Goerli
   421613: 3, // Arbitrum Goerli
   84531: 6, // Base Goerli
-  80001: 7, // Polygon Mumbai
+  80002: 7, // Polygon Amoy
```
Comment (james-a-morris) [inline deploy/consts.ts:281]: Nit: we may want to keep the entry here (like we do with eth goerli & base goerli) - we should also probably add eth sepolia

**Review by james-a-morris — APPROVED** [review]
+1

**Comment by bmzig** [comment]
> +1 Have we run the hubpool commands to hook this up?

I've only added pool rebalance routes and set the adapter, so I'm yet to update the config store and deposit routes.

**Pull request merged**